recipes/pyrad_arm/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "pyrad_arm" %}
+{% set pip_name = "pyrad_mch" %}
```

Comment:
Is there a reason why the name of the package is different than the pypi name?

Reply:
if this name is only used in the url, maybe enbed it there and don't have this variable? Also since it seems like you develop on GitHub, can you grab the release tarball from there?